src/axom/inlet/Inlet.hpp

```diff
@@ -260,6 +260,18 @@ class Inlet
     return m_globalContainer.contains(name);
   }
 
+  /*!
+   *****************************************************************************
+   * \brief Return whether a subobject with the given name was provided by the user
+   *
+   * \see Container::isUserProvided
+   *****************************************************************************
+   */
+  bool isUserProvided(const std::string& name) const
```

Comment:
minor: Should this be `hasUserProvided(name)` instead of `isUserProvided(name)` ?

I.e.  the full call on a container is `container.isUserProvided(name)`, but it seems like you're raelly asking if the container **has** a user provided subobject named `name`

Reply:
The current functionality is `hasUserProvided()` (no args) on a `Container` or `Field` which checks the calling object - I agree that semantically `hasUserProvided(string name)` called with a subobject name makes more sense, but I'm not sure if it would be confusing to have two methods with names that are so similar.